ship.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   beforeCommitChanges: ({ exec }) => {
     exec('yarn doctoc');
   },
-  pullRequestReviewer: ['@algolia/instantsearch-for-websites'],
+  pullRequestTeamReviewer: ['instantsearch-for-websites'],
```

Comment:
Side note: since it's an array I would expect the option to be named `pullRequestTeamReviewers`.

Reply:
Thanks for the feedback. It started as a string at first, so its name was singular. Now that it supports array, I should deprecate the old name and make a new name for later version.